ansible_files/roles/equinix/export_primary_device_connection_details/templates/inventory.j2

```diff
@@ -0,0 +1,17 @@
+{# The CI expects the node that runs test-infra to be in the "primary" group -#}
+{# Other nodes are ungrouped, and will be targeted when the playbook runs on "all" nodes -#}
+
+{%- set dump_vars=["ansible_host", "ansible_user", "ansible_ssh_private_key_file"] -%}
+{% macro build_host(hostname) -%}
+    {{ hostname -}}
+    {% for var in dump_vars %}
+        {{- " " + var }}={{ hostvars[hostname][var] }}
+    {%- endfor %}
+{% endmacro -%}
```

Comment:
a bit too dark-magicy to me
maybe doing it by repeating the items can give it a bit of a clearer structure (and let future manipulations be more straight-forward)

Reply:
yes, looks great!
thank you!